src/dataset-builder/DatasetBuilder.ts

```diff
@@ -541,25 +547,25 @@ export const DatasetBuilderContents = ({
           h(CohortSelector, {
             cohorts,
             selectedCohorts,
-            onChange: setSelectedCohorts,
+            onChange: updateSelectedCohorts,
```

Comment:
What is the difference between `set` and `update`? I would expect `set` to overwrite and `update` to modify but it looks like `update` is the same as `set` here, so I'm not sure why the name was changed.

Reply:
I like preserving `set` for specifically state setters. In this case `set` describes the linked function for the state value, but `update` is used once it is no longer tied to a piece of the component's state.